README.md

```diff
@@ -141,10 +142,19 @@ The shared storage worklet invocation methods (`addModule`, `run`, and `selectUR
         *   `keepAlive` (defaults to false), a boolean denoting whether the worklet should be retained after it completes work for this call.
             *   If `keepAlive` is false or not specified, the worklet will shutdown as soon as the operation finishes and subsequent calls to it will fail.
             *   To keep the worklet alive throughout multiple calls to `run()` and/or `selectURL()`, each of those calls must include `keepAlive: true` in the `options` dictionary.
+*   `window.sharedStorage.run(name, options)`,  \
+`window.sharedStorage.selectURL(name, urls, options)`, …
+    *   The behaivor is identical to `window.sharedStorage.worklet.run(name, options)` and `window.sharedStorage.worklet.selectURL(name, urls, options)`.
+*   `window.sharedStorage.createWorklet(url, options)`
+    *   Creates a new worklet, and loads and adds the module to the worklet (similar to the handling for `window.sharedStorage.worklet.addModule(url, options)`).
+    *   The worklet uses the `url`'s origin as its partition origin for accessing shared storage data and for budget checking and withdrawing.
+    *   The object that the returned Promise resolves to has the same type with the implicitly constructed `window.sharedStorage.worklet`. However, for worklet created via `window.sharedStorage.createWorklet(url, options)`, only `selectURL()` and `run()` are legitimate, whereas `addModule()` will throw an error. This is to prevent leaking shared storage data via `addModule()`, similar to the reason why `addModule()` can only be invoked once on the implicitly constructed `window.sharedStorage.worklet`.
```

Comment:
s/for worklet created via/for a worklet created via/

Suggest: ..."only `selectURL()` and `run()` are available, whereas calling `addModule()` will throw an error."